tests/samplers_tests/test_samplers.py

```diff
@@ -549,3 +550,27 @@ def after_trial(
     study.tell(study.ask(), 1.0)
 
     assert n_calls == 1
+
+
+@parametrize_sampler
+def test_sample_single_distribution(sampler_class: Callable[[], BaseSampler]) -> None:
+
+    relative_search_space = {
+        "a": UniformDistribution(low=1.0, high=1.0),
+        "b": LogUniformDistribution(low=1.0, high=1.0),
+        "c": DiscreteUniformDistribution(low=1.0, high=1.0, q=1.0),
+        "d": IntUniformDistribution(low=1, high=1),
+        "e": IntLogUniformDistribution(low=1, high=1),
+        "f": CategoricalDistribution([1]),
+    }
+
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", optuna.exceptions.ExperimentalWarning)
+        sampler = sampler_class()
+    study = optuna.study.create_study(sampler=sampler)
+
+    for _ in range(100):
```

Comment:
By chance, I found that the introduced test function `test_sample_single_distribution` is time-consuming. On the master branch, `pytest tests/samplers_tests/test_samplers.py` takes about `8.45s` to run, but `test_sample_single_distribution` takes `24.01s` (so the total will be about `32s`).

How about reducing the number of epochs?

```bash
# master
pytest tests/samplers_tests/test_samplers.py
====================================================== 164 passed, 70 warnings in 8.45s =======================================================

# this branch
pytest tests/samplers_tests/test_samplers.py
===================================================== 171 passed, 368 warnings in 32.29s ======================================================

# more specifically...
pytest tests/samplers_tests/test_samplers.py::test_sample_single_distribution
====================================================== 7 passed, 298 warnings in 24.01s =======================================================
```

Reply:
Thanks for the pointing. I reduce the number of trials from 100 to 10.